temporalcli/commandsgen/commands.yml

```diff
@@ -1400,8 +1400,17 @@ commands:
   - name: temporal server
     summary: Run Temporal Server
     description: |
-      Run a development Temporal Server on your local system. View the Web UI for
-      the default configuration at http://localhost:8233:
+      Run a development Temporal Server on your local system.
+
+      +------------------------------------------------------------------------+
+      | WARNING: The development server is not intended for use in production. |
+      | HTTP security checks have been relaxed to allow for easier local use.  |
+      |                                                                        |
+      | If you need a production-quality Temporal Service, please see:         |
+      | https://docs.temporal.io/production-deployment                         |
+      +------------------------------------------------------------------------+
+
+      View the Web UI for the default configuration at http://localhost:8233:
```

Comment:
nit on pre-existing code: let's lose the terminal colon here. It's not helping anyone much and could well be an annoyance in a terminal emulator with a bad url-recognizing regex.